src/test/java/teammates/test/cases/ui/browsertests/AdminSearchPageUiTest.java

```diff
@@ -74,6 +78,10 @@ private void testSearch() {
         assertTrue(isSearchPanelPresent());
         assertTrue(isSearchDataDisplayCorrect());
         
+        StudentAttributes student = testData.students.get("student1InCourse1");
+        InstructorAttributes instructor = testData.instructors.get("instructor1OfCourse1");
```

Comment:
I think this test fails on master because the student's name links to the student's record page using the id of the instructor with the 'Helper' role. However, the test data is incorrect. The data of other tests have this problem as well.